netbench/cdk/src/main/java/com/aws/NetbenchAutoApp.java

```diff
@@ -65,22 +65,49 @@ public static void main(final String[] args) {
         
         // Stack instantiation
         ReportStack reportStack = new ReportStack(app, "ReportStack", StackProps.builder()
-            .env(makeEnv(awsAccount, clientRegion))
+            .env(makeEnv(awsAccount, serverRegion))
             .build());
 
-        ClientStack clientStack = new ClientStack(app, "ClientStack", ClientStackProps.builder()
+        ClientServerStack clientStack = new ClientServerStack(app, "ClientStack", ClientServerStackProps.builder()
             .env(makeEnv(awsAccount, clientRegion))
             .bucket(reportStack.getBucket())
             .instanceType(ec2InstanceType)
+            .cidr("10.0.0.0/16")
+            .stackType("client")
             .build());
 
-        ServerStack serverStack = new ServerStack(app, "ServerStack", StackProps.builder()
+        ClientServerStack serverStack = new ClientServerStack(app, "ServerStack", ClientServerStackProps.builder()
             .env(makeEnv(awsAccount, serverRegion))
+            .bucket(reportStack.getBucket())
+            .instanceType(ec2InstanceType)
+            .cidr("11.0.0.0/16")
```

Comment:
where does this come from (and `10.0.0.0/16`)?

Reply:
10.0.0.0/16 is just the default value for the VPC's. However, since the CIDR values of the client and server must be different, I changed the server value to 11.0.0.0/16. I don't think the exact value matters as long as they don't overlap.